.contentful.json.sample

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
What purpose does `contentful.json` serve?

Reply:
I just remove it